tla-types/src/test/scala/at/forsyte/apalache/tla/types/TestTypesApi.scala

```diff
@@ -0,0 +1,42 @@
+package at.forsyte.apalache.tla.types
```

Comment:
This is more or less a redundant test suite, given the tests in `typecmp`.

Reply:
I agree that it is not as exhaustive. I have introduced this test as a minimal test of the API, in contrast to the exhaustive tests in `typecmp`.